config/src/config/execution_config.rs

```diff
@@ -12,6 +12,8 @@ use std::{
     sync::Arc,
 };
 
+const GENESIS_DEFAULT: &str = "genesis.blob";
```

Comment:
btw shouldn't all these default files be sorted in `test_config` / `template` / `genesis` folders?